StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.inf

```diff
@@ -50,4 +54,4 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdFixedCommBufferPages
 
 [Depex]
-  TRUE
+  gEfiPeiMpServicesPpiGuid

```

Comment:
could you move the dependency to the commit "StandaloneMmPkg/MmIpl: Create HobList for Standalone MM foundation". I guess this dependency is needed because the guid HOB gMpInformation2HobGuid